### PR TITLE
Clean repo

### PR DESCRIPTION
**Pull request by ethanhung11**
Removing unused files & combining where possible. Branch is behind main & does not the ALS/CLS line search changes.

### CODE REVIEW

**Review by aarmey — CHANGES_REQUESTED** [review]
Looks good. Can you handle the merge conflicts?

**Comment by ethanhung11** [comment]
Should be fixed now!

File: timpute/figures/common.py [hunk 1]
```diff
@@ -6,6 +6,7 @@
 import matplotlib
 from matplotlib import gridspec, pyplot as plt
 from matplotlib.lines import Line2D
+import svgutils.transform as st
```
Comment (aarmey) [inline timpute/figures/common.py:9]: This is unused. Remove.

File: timpute/figures/figure3.py [hunk 2]
```diff
@@ -1,7 +1,6 @@
 import numpy as np
-from .runImputation import *
-from ..plot import *
-from ..common import *
+from .figure_helper import *
```
Comment (aarmey) [inline timpute/figures/figure3.py:2]: Please convert all of these to non-wildcard imports. Wildcard imports can cause many issues due to unintentional imports.

**Review by aarmey — CHANGES_REQUESTED** [review]
A couple more things.

File: timpute/figures/figure3.py (hunk 2 above)
Comment (ethanhung11) [reply timpute/figures/figure3.py]: should be fixed now

**Review by aarmey — APPROVED** [review]
Great! Merging.

**Pull request merged**